test_pytest_rerunfailures.py

```diff
@@ -388,3 +388,13 @@ def pytest_runtest_teardown(item):
             assert item.execution_count == 3""")
     result = testdir.runpytest('--reruns', '2')
     assert_outcomes(result, passed=3, rerun=2)
+
+def test_rerun_report(testdir):
+    testdir.makepyfile('def test_pass(): assert False')
+    testdir.makeconftest("""
+        def pytest_runtest_logreport(report):
+            assert hasattr(report, 'rerun')
+            assert isinstance(report.rerun, int)
```

Comment:
Maybe we could assert the upper limit for reruns here. 
```suggestion
            assert isinstance(report.rerun, int)
            assert report.rerun <= 2
```